bareos/bconsole.sls

```diff
@@ -24,13 +24,13 @@ bareos_bconsole_cfg_file:
     - name: {{ bareos.config_dir }}/{{ bareos.bconsole.config_file }}
     - source: salt://bareos/files/bareos-config.jinja
     - context:
-        config: {{ bc_config|json() }}
+        config: {{ bc_config|yaml() }}
         default_password: {{ bareos.default_password }}
         require_password: {{ require_password }}
     - template: jinja
-    - mode: 644
-    - user: root
-    - group: root
+    - mode: 750
```

Comment:
Why `750` for a config file? For hardening config files, I'd say `640 (rw-r-----)` would be better?